### ISSUE TITLE
TestCafe: Error: Cannot find module 'C:Users...testControllerHolder.js'

### ISSUE THREAD

**Issue by hubidu** [Codeception/CodeceptJS#1765]
#### What are you trying to achieve?

I want to use the TestCafe helper on Windows.

#### What do you get instead?

I get the above error message

### Details

* CodeceptJS version: 2.2.0
* NodeJS Version: 10.x
* Operating System: Windows
* TestCafe

**Comment by hubidu** [comment]
Line 12 of codeceptjs/lib/helper/testcafe/testcafe-utils.js needs to replaced with this to work on Windows:

```
  const testControllerHolderDir = __dirname.replace(/\\/g, '/');
```

**Comment by DavertMik** [comment]
Fixed by #1769
